readers/cassandra/messages_test.go

```diff
@@ -68,9 +89,9 @@ func TestReadAll(t *testing.T) {
 		},
 		"read message last page": {
 			chanID:   chanID,
-			offset:   40,
-			limit:    10,
-			messages: messages[40:42],
+			offset:   0,
+			limit:    15,
+			messages: messages,
 		},
```

Comment:
You've removed test cases that are testing offset that is greater than 0. Please add them.